atd-vze/src/views/Fatalities/fatalityGridTableParameters.js

```diff
@@ -68,3 +71,202 @@ export const fatalityGridTableColumns = {
+            },
+          ],
+        },
+      },
+      {
+        id: "cyclist",
+        label: "Cyclist Fatality",
+        filter: {
+          where: [
+            {
+              "_or: [ { person: { unit: { unit_desc_id: { _eq: 3 } } } }, { primaryperson: { unit: { unit_desc_id: { _eq: 3 } } } } ]": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "pedestrian",
+        label: "Pedestrian Fatality",
+        filter: {
+          where: [
+            {
+              "_or: [ { person: { unit: { unit_desc_id: { _eq: 4 } } } }, { primaryperson: { unit: { unit_desc_id: { _eq: 4 } } } } ]": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "scooter",
+        label: "Scooter-rider Fatality",
+        filter: {
+          where: [
+            {
+              "_or: [ { _and: [ { person: { unit: { unit_desc_id: { _eq: 177 } } } }, { person: { unit: { veh_body_styl_id: { _eq: 177 } } } } ] }, { _and: [ { primaryperson: { unit: { unit_desc_id: { _eq: 177 } } } }, { primaryperson: { unit: { veh_body_styl_id: { _eq: 177 } } } } ] } ]": null,
+            },
+          ],
+        },
+      },
+    ],
+  },
+  groupStatus: {
+    icon: "newspaper-o",
+    label: "Status",
+    filters: [
+      {
+        id: "status_open_short_term",
+        label: "Open - Short Term",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 1 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_open_long_term",
+        label: "Open - Long Term",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 2 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_closed_no_action_rec",
+        label: "Closed - No Action Recommended",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 3 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_closed_rec_implemented",
+        label: "Closed - Recommendation Implemented",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 4 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_closed_no_action_taken",
+        label: "Closed - No Action Taken",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 5 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_null",
+        label: "Null",
+        filter: {
+          where: [
+            {
+              "_not: { recommendation: {recommendation_status_id: { _is_null: false } } }": null,
```

Comment:
if i were to leave out the `not` and just check for records that have null recommendation statuses this filter wouldn't capture fatalities that dont have a recommendation associated with them at all yet.